docs/getting-started/publish_to_pypi.md

```diff
@@ -3,8 +3,10 @@
 ## Differences in the process
 Now that we've published to test pypi. PyPI should be cake! Instead of re-writing the process with a few tweaks I'll mention the main differences here for getting to PyPI.
 
-1. Visit [PyPI](https://www.pypi.org) (not test pypi) to create an account
-2. Set the given username and password as `PYPI_USERNAME` and `PYPI_PASSWORD`.
+1. Visit [PyPI](https://www.pypi.org) (not test pypi) to create an account.
```

Comment:
This change is not correct, I think. I am not sure if it worked for you with only the token, but the Pypi documentation clearly states to use `PYPI_USERNAME="__token__"`